app/assets/stylesheets/base/decorations.scss

```diff
@@ -0,0 +1,73 @@
+@import '../vars/variables';
+
+// Opacity
+@include generate-classes(
+  'opacity',
+  'opacity',
+  (
+    '100': 1,
+    '75': .75,
+    '50': .50,
+    '25': .25,
+    '0': 0,
+  )
+);
+
+// Box Shadow
+.shadow-none {
+  box-shadow: none !important; 
+}
+
+// Cursor
+@include generate-classes(
+  'cursor',
+  'cursor',
+  (
+    'auto': auto,
+    'default': default,
+    'pointer': pointer,
+    'wait': wait,
+    'text': text,
+    'move': move,
+    'not-allowed': not-allowed,
+  )
+);
+
+// Outline
+.outline-none {
+  outline: none !important;
+}
+
+// Pointer events
+@include generate-classes(
+  'pointer-events',
+  'pointer-events',
+  (
+    'auto': auto,
+    'none': none,
+  )
+);
+
+// Screen reader only
+.sr-only {
```

Comment:
OK, so `!important` seems to be in a lot of places. Is this due to our existing CSS being too specific?

Reply:
Hah, sorry I replied in other comment: https://github.com/thepracticaldev/dev.to/pull/6334#discussion_r388306838

[Yes and no. ](https://tailwindcss.com/docs/configuration/#important)

- NO if our frontend was ONLY tailwind (in other words, if we were starting from scratch)
- YES if we have existing frontend codebase and if we also have components defined.

I know it may feel dirty but it makes perfect sense IMO.